File: FCUBEAPI/FleetTrans/Models/SparesHistoryRptListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FleetTrans.Models
{
    public class SparesHistoryRptListModel
    {
        public List<SparesHistoryRptModel> SparessHistoryRptList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}

File: FCUBEAPP/src/app/models/spareshistoryrptlistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Spareshistoryrptmodel } from "./spareshistoryrptmodel";

export class Spareshistoryrptlistmodel {
    sparessHistoryRptList: Spareshistoryrptmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}
